stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -939,6 +939,21 @@ public class ChatClient internal constructor(
             updateMessage,
         )
 
+    @CheckResult
+    public fun updateChannelPartial(
```

Comment:
We should add kdoc for these new public methods, it could explain what this method does vs the regular update, and what `set` an `unset` are